spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/ehcache/EhCacheProperties.java

```diff
@@ -0,0 +1,57 @@
+package org.springframework.boot.autoconfigure.ehcache;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Configuration properties to configure {@link org.springframework.cache.ehcache.EhCacheFactoryBean}.
+ *
+ * @author Eddú Meléndez
+ * @since 1.3.0
+ */
+@ConfigurationProperties(prefix = "ehcache", ignoreUnknownFields = false)
+public class EhCacheProperties {
```

Comment:
We shouldn't use top level namespace like that for configuration. Let's go for `spring.cache.ehcache`.